package-examples/ghost/ghost/setlabels.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: fn.kpt.dev/v1alpha1
+kind: SetLabels
+metadata:
+  name: ghost
+  annotations:
+    config.kubernetes.io/local-config: "true"
+labels:
```

Comment:
😞 no spec. I think all custom functionConfig type do not follow the `spec` `status` rule. The argument is that those resources won't be deployed to the cluster so it is not a strict spec/status. Skipping spec saves some manual editing. 

I'm more happy to add the `spec` to `functionConfig`, because it does help answering questions like "what fields & resources a user could be interested in modifying"  https://kubernetes.slack.com/archives/C0155NSPJSZ/p1657561360040189